lib/ReactViews/Generic/Editor.tsx

```diff
@@ -4,6 +4,7 @@ import { Editor } from "@tinymce/tinymce-react";
 import PropTypes from "prop-types";
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import tinymce from "tinymce"; // must import despite being unused
+import type { Editor as TinyMCEEditor } from "tinymce";
```

Comment:
I never understood why tinymce must be imported despite being unused. Can the previous line be removed now that something else is imported from tinymce?

Reply:
I didn't check that, but type imports will be removed in compile time so those new imports doesn't change anything